### PR TITLE
Add profiler class

### PR DESCRIPTION
**Pull request by mbeitchman**
This commit adds the first class and unit test to the project.

Please make sure your commit messages follow this guidance:

1. Separate subject from body with a blank line
2. Limit the subject line to 50 characters
3. Capitalize the subject line
4. Do not end the subject line with a period
5. Use the imperative mood in the subject line
6. Wrap the body at 72 characters
7. Use the body to explain what and why vs. how

Please make sure to include unit tests for all commits.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mbeitchman** [comment]
@awslabs/emr-workload-profiler @baleksan @hadesmeza can I get a code review? =)

**Comment by mbeitchman** [comment]
Hi all,

Looking forward to your feedback on the first pr. I want it to be educational in the sense that we can have new contributors read the commit message to get an understanding of the standards for the project.

thx,
Marc

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            <version>2.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>6.11</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
```
Comment (baleksan) [inline pom.xml]: Any reason not to move to latest mockito - https://mvnrepository.com/artifact/org.mockito/mockito-core/2.13.0. Might as well get the greatest?

File: src/test/java/com/aws/emr/profiler/core/ProfilerTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ */
+package com.aws.emr.profiler.core;
+
+import org.testng.annotations.Test;
+
+public class ProfilerTest {
+    @Test
+    public void firstTest() {
+        Profiler p = new Profiler();
```
Comment (baleksan) [inline src/test/java/com/aws/emr/profiler/core/ProfilerTest.java]: You might as well add a example method to the profiler class and demonstrate assertion use in junit
```java
public class Profiler {
      public int multiplyByTwo(int base) {
               return base * 2;
      } 
}
```
and then 

```java
@Test
public void firstTest() {
           Profiler p = new Profiler();
           assertEquals(p.multiplyByTwo(2), 4);
          assertEquals(p.multiplyByTwo(0), 0);
}
```

File: pom.xml (hunk 1 above)
Comment (mbeitchman) [reply pom.xml]: No reason. I will update the pr.

File: src/test/java/com/aws/emr/profiler/core/ProfilerTest.java (hunk 2 above)
Comment (mbeitchman) [reply src/test/java/com/aws/emr/profiler/core/ProfilerTest.java]: Makes sense. I think I can add something simple so the test will assert something.

**Pull request merged**